packages/blocks/src/elements.spec.ts

```diff
@@ -112,4 +136,110 @@ describe('Slack Element widgets', () => {
       }),
     ).toMatchSnapshot();
   });
+
+  it('renders a minimal conversation select input element', () => {
+    expect.assertions(1);
+    expect(
+      ConversationsSelectInputElement('channel', 'Select a channel'),
```

Comment:
Minor: Change to 'conversation' here and in next two tests (as it's different from ChannelsSelectInputElement).